requirements.txt

```diff
@@ -1,16 +1,24 @@
 amqp==5.2.0
+amqp==5.2.0
 bcrypt==3.2.0
+beautifulsoup4==4.12.3
+billiard==4.2.1
 billiard==4.2.1
 blinker==1.4
 Bottleneck==1.3.7
 celery==5.4.0
+celery==5.4.0
 certifi==2024.8.30
 cffi==1.14.6
 charset-normalizer==3.4.0
 click==8.1.7
 click-didyoumean==0.3.1
 click-plugins==1.1.1
 click-repl==0.3.0
+click==8.1.7
+click-didyoumean==0.3.1
+click-plugins==1.1.1
+click-repl==0.3.0
```

Comment:
repeated again. recommend to read through this file to check repetition

Reply:
fixed